pom.xml

```diff
@@ -26,6 +26,18 @@
   <description>Cassandra extensions for Quarkus</description>
   <url>https://quarkus.io/guides/cassandra</url>
   <inceptionYear>2020</inceptionYear>
+  <distributionManagement>
```

Comment:
Not sure what this is for?

Reply:
Not supposed to go here. 👍 Will be removed in a second.